test/SourceKit/CodeComplete/complete_local_decl.swift

```diff
@@ -0,0 +1,26 @@
+// rdar://128294522 - Make sure we don't generate USRs for decls in local contexts.
+
+func test1() {
+  let loclVar = 0 // This typo is intentional; otherwise mangling applies a substitution for 'local'.
+  // RUN: %sourcekitd-test -req=complete -pos=%(line):3 %s -- %s | %FileCheck %s --check-prefix LOCAL_VAR --implicit-check-not='key.associated_usrs: "s:{{.*}}loclVar{{.*}}"'
```

Comment:
Would a `LOCAL_VAR-NOT` line be clearer than `--implicit-check-not`? Or is there a reason why we can’t use `CHECK-NOT`?

Reply:
Mainly just avoids relying on the `CHECK-NOT` happening before or after the match. Probably not that big of a deal here because we enforce a certain ordering, but makes it a bit more future proof 🤷

Reply:
Switched to using explicit `CHECK-NOT`s, looking at it again the implicit `CHECK-NOT`'s weren't really saving many lines anyway